services/web/server/src/simcore_service_webserver/workspaces/_groups_db.py

```diff
@@ -110,9 +119,9 @@ async def get_workspace_group(
         )
     )
 
-    async with get_database_engine(app).acquire() as conn:
+    async with pass_or_acquire_connection(get_asyncpg_engine(app), connection) as conn:
         result = await conn.execute(stmt)
-        row = await result.first()
+        row = result.first()
```

Comment:
And this one. I would expect this one to be await maybe the `execute` above should be a `stream`?